src/features/courses/course-controller.ts

```diff
@@ -4505,24 +4543,33 @@ You can contact your student at ${options.student.email} or by replying to this
                             as: 'studentGradeInstance',
                             attributes: ['id', 'webworkQuestionPath'],
                             where: {
-                                active: true,
+                                active: false,
```

Comment:
Why do we want active false?

Reply:
Good catch, this was meant to be `required: false`.